driver/CMakeLists.txt

```diff
@@ -27,6 +27,10 @@
 find_package(Threads REQUIRED)
 
 add_executable(MIOpenDriver main.cpp InputFlags.cpp)
+if(WIN32)
+    # Refer to https://en.cppreference.com/w/cpp/language/types for details.
+    target_compile_options(MIOpenDriver PRIVATE $<BUILD_INTERFACE:$<$<CXX_COMPILER_ID:Clang>:-U__LP64__>>)
```

Comment:
@apwojcik Can you please suggest why this is needed?

Please also note that
```bash
$ /opt/rocm/llvm/bin/clang -dM -E -x c /dev/null | grep LP64
#define _LP64 1
#define __LP64__ 1
```
so maybe we need to undef _LP64 as well.